script/e2e-test.sh

```diff
@@ -16,7 +16,7 @@ IMG_MODIFIER=${4:-""}
 TEST_TIMEOUT_MINUTES=${5:-"4"}
 DEX_IP=${6:-"172.18.0.2"}
 ADDITIONAL_CLUSTER_IP=${7:-"172.18.0.3"}
-KAPP_CONTROLLER_VERSION=${8:-"v0.32.0"}
+KAPP_CONTROLLER_VERSION=${8:-"v0.41.2"}
```

Comment:
So the version is not being passed through and instead we are using this hardcoded, fallback version, no?

Reply:
Currently yes, we should work in providing the right values from CircleCI to `e2e-test.sh`.